database/migrations/2023_06_10_220139_create_api_logs.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        if (!env('DB_LOGS_DATABASE', false)) {
+            return;
+        }
+        Schema::connection('logs')->create('api_logs', function (Blueprint $table) {
+            $table->id();
+            $table->timestamps();
+            $table->unsignedInteger('user_id')->nullable();
+            $table->unsignedInteger('campaign_id')->nullable();
+            $table->text('uri')->nullable();
+            $table->json('params')->nullable();
```

Comment:
I forgot that we need an index on `created_at` since we're using it in the `WHERE` for deletion :(